pkg/client/open_windows.go

```diff
@@ -0,0 +1,19 @@
+// +build windows
+
+package client
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+var (
+	cmd      = "url.dll,FileProtocolHandler"
+	runDll32 = filepath.Join(os.Getenv("SYSTEMROOT"), "System32", "rundll32.exe")
+)
+
+func open(input string) *exec.Cmd {
+	cmd := exec.Command(runDll32, cmd, input)
```

Comment:
This is really a no-go.

1) You are not even checking if this file exists at all
2) What about Windows 64bit users? This will result in an fatal error...

Reply:
1. `exec.Command()` just sets the stage. Trying to execute a non existing binary by calling `cmd.Run()` or `cmd.Start()`, which is done later in the code would result in an error that is treated as fatal in https://github.com/owncloud/ocis/pull/93/files/e35221c0f5398a46073879034ebfd33faffc6236#diff-fb87476fa66849c8514ba700e715b4b6R96
2. every Windows64 bit installations contains the 32bit subsystem, which is known as WOW64 ... Windows on Windows64. I think it is safe to assame it is present in every Windows 10 installation out there.